Samples/MobileBuyIntegration/MobileBuyIntegration/CartManager.swift

```diff
@@ -366,9 +357,7 @@ class CartManager: ObservableObject {
                 throw Errors.invariant(message: "cart returned nil")
             }
 
-            DispatchQueue.main.async {
-                self.cart = cart
-            }
+            self.cart = cart
```

Comment:
Would this be supported?


```suggestion
            let {cart} = self;
```

Reply:
No, that's not a pattern that's available in swift